draft-wqk-opsawg-control-schedule-framework.md

```diff
@@ -511,11 +512,69 @@ In the context of a tidal network, If the network maintains all the devices
 up to guarantee the maximum throughput all the time, a lot of power will be
 wasted. The energy-saving methods may include the deactivation of some or all
 components of network nodes. These activities have the potential to alter
-network topology and impact data routing in a variety of ways.  Ports on
+network topology and impact data routing in a variety of ways.  Interfaces on
 network nodes can be selectively disabled or enabled based on traffic patterns,
 thereby reducing the energy consumption of nodes during periods of low network
 traffic.
 
+The network management devices need to control the state of interfaces of each node
+by the ietf-tvr-node model defined in {{?I-D.ietf-tvr-schedule-yang}}.
+
+The following indicates the example of a scheduling node interface that enabled at 7:00 AM and disabled at 1:00 AM, every day, from December 1, 2025 in UTC.
+The JSON encoding is used only for illustration purposes.
+
+~~~~
+{
+    "ietf-tvr-node:node-schedule": [
+        {
+            "node-id": 12345678,
+            "node-power-schedule": {
+              "power-default": true
+              "schedules": [
+                {
+                  "schedule-id": 111111,
+                  "period-start": "2025-12-01T00:00:00Z",
+                  "period-end": "2026-12-01T07:00:00Z"
+                }
+              ]
+            }
+            "interface-schedule": [
+              {
+                "name": "interace1",
+                "default-available": false,
+                "default-bandwidth": 1000000000,
+                "attribute-schedule": {
+                  "schedules":[
+                    {
```

Comment:
suggest to remove the first schedule instance, which is contradictory with the second one.